resolvers.js

```diff
@@ -77,6 +77,15 @@ function baseDatasetQuery() {
   });
 }
 
+function checkFetchRes(resp) {
+  if (resp.status >= 200 && resp.status < 300) {
```

Comment:
I'd use `resp.ok` flag, as it has the same meaning as per MDN

Reply:
@iprotsyuk that's interesting case, what I found with fetch is that resp will be ok even if 404 status is raised. e.g. when one sends a full url , so this was the only way to catch this that I came up with
More:
https://github.com/github/fetch/issues/155

Reply:
actually I checked it once again and seems that resp.ok works fine too. Don't know what was the thing before